.github/workflows/unit-tests.yml

```
@@ -0,0 +1,24 @@
+name: Unit tests
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [main]
+    types: ready_for_review
+
+env:
+    PHARMAPKGS_REMOTE_REPO: https://raw.githubusercontent.com/r-hub/repos/main
+    PHARMAPKGS_LIMIT: 5
+
+jobs:
+    main:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v2
```

v2 is no [longer supported](https://github.com/actions/checkout) and I'm not sure it works well on all machines:

```suggestion
        - uses: actions/checkout@v4
```